core/server/helpers/get.js

```diff
@@ -184,6 +191,22 @@ get = function get(resource, options) {
         logging.error(err);
         data.error = err.message;
         return options.inverse(self, {data: data});
+    }).finally(function () {
+        const end = new Date();
+        const totalMs = end.valueOf() - start.valueOf();
```

Comment:
`Date.now()` means you don't have to f w/ valueOf - don't bother changing if you're not gonna edit this file